Makefile

```diff
@@ -0,0 +1,42 @@
+all: buster ubuntu
+
+buster_base:
+	docker build -f "build/Dockerfile.base.buster" -t indy-node-container/base:buster build
+
+buster: buster_base
+	docker build -f "build/Dockerfile.buster" -t indy-node-container/indy_node:buster --build-arg BASE=python:3.6-slim-buster --build-arg BUILDER_BASE=indy-node-container/base:buster build
+
+ubuntu_base:
+	docker build -f "build/Dockerfile.base.ubuntu18" -t indy-node-container/base:ubuntu build
+
+ubuntu: ubuntu_base
+	docker build -f "build/Dockerfile.ubuntu18" -t indy-node-container/indy_node:ubuntu --build-arg BASE=ubuntu:18.04 --build-arg BUILDER_BASE=indy-node-container/base:ubuntu build
+
+clean_buster_base:
+	-docker image rm indy-node-container/base:buster
```

Comment:
Why do we use here `-<COMMAND>` is this something related to Makefiles?  Or only a typo ?

Reply:
the `-` means: "continue on error/non-zero exit", which occurs if the image is missing

Reply:
Ah that's cool. I used mostly `COMMAND || /bin/true`